src/modules/common/components/CopyToClip/CopyToClip.tsx

```diff
@@ -0,0 +1,73 @@
+import { IconButton, makeStyles, Theme, Tooltip, Typography } from '@material-ui/core';
+import { truncateWalletAddr } from 'modules/common/utils/truncateWalletAddr';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { t } from 'modules/i18n/utils/intl';
+import { CopyIcon } from 'modules/common/components/Icons/CopyIcon';
+import { DoneIcon } from 'modules/common/components/Icons/DoneIcon';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  addressText: {
+    color: theme.palette.text.secondary,
+    fontSize: 14,
+  },
+  clipboardBtn: {
+    fontSize: 20,
+    border: 'none !important',
+    marginLeft: theme.spacing(1),
+    color: theme.palette.text.secondary,
+  },
+  clipboardBtnIcon: {
+    fontSize: 'inherit',
+  },
+  clipboardBtnIconDone: {
+    color: theme.palette.success.light,
+  },
+}))
```

Comment:
It would be great if the styles were placed in a separate file, in accordance with the code style in the project.